app/src/main/java/com/example/mob_lab1/MainActivity.java

```diff
@@ -0,0 +1,93 @@
+package com.example.mob_lab1;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+
+public class MainActivity extends AppCompatActivity {

```

Comment:
не коректна назва, activity має називатись відповідно до того, що на ньому відбувається;
```suggestion
public class SignUpActivity extends AppCompatActivity {
```